docs/ios/xcsync.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Xcode Sync (xcsync) for .NET MAUI
+description: [Xcsync is designed to provide .NET developers with improved support for editing Apple specific files]
+author: haritha-mohan, mcumming
+ms.date: [08/05/2024]
+---
+# Xcode Sync (Xcsync)
+
+**This article applies to:** ✔️ .NET Core 9 Preview 7 SDK and later versions
+
+Xcode Sync (Xcsync) is a tool that enables developers to leverage Xcode for managing Apple specific files with .NET projects. The tool generates a temporary Xcode project from a .NET project and synchronizes changes to the Xcode files back to the .NET project.
+
+Supported file types include:
+
+- Asset Catalog
+- Plist
+- Storyboard
+- Xib
+
+The tool has 2 commands: generate and sync. Use generate for tracking your changes from your .NET to Xcode and sync for tracking your changes from Xcode to .NET. You can also add the xcsync tool path to your PATH environment variable to invoke the tool directly as a more seamless dotnet command.
+
+## Synopsis
+
+### xcsync-generate
+
+```dotnetcli
+dotnet build /t:xcsync-generate [-p|--project <PROJECT>]
+    [-tfm|--target-framework-moniker <FRAMEWORK>]
+    [-t|--target <TARGET_DIRECTORY>] [-f|--force [true|false]]
+    [-o|--open [true|false]]
+    [-v|--verbosity <LEVEL>]
+    [-d|--dotnet-path <.NET_SDK_PATH>]
+
+dotnet build /t:xcsync-generate -?|-h|--help
+```
+
+### xcsync-sync
+
+```dotnetcli
+dotnet build /t:xcsync-sync [-p|--project <PROJECT>]
+    [-tfm|--target-framework-moniker <FRAMEWORK>]
+    [-t|--target <TARGET_DIRECTORY>]
+    [-v|--verbosity <LEVEL>]
+    [-d|--dotnet-path <.NET_SDK_PATH>]
+
+dotnet build /t:xcsync-sync -?|h|--help
+```
+
+### Arguments
+
+- **`-p|--project <PROJECT>`**
+  The .NET project file to build. This is **required**.
+
+### Options
+
+These are optional; if not specified, the default value will be used.
+
+- **`-tfm|--target-framework-moniker <FRAMEWORK>`**
+  Invoke the tool for a specific framework. The framework must be defined in the project file. Examples: `net9.0-ios`, `net9.0-maccatalyst`. **Required** if the .NET project supports multiple tfms (for example, a standard MAUI project). If a single platform project, the default value will be the single tfm specified in the project file.
```

Comment:
eh this is not as clean as i would like to be...

but basically it's required when there's multiple tfms to choose from but not for when there is only one..the tool does a decent job of error-handling this case...